daml/Marketplace/Transfer.daml

```diff
@@ -6,7 +6,7 @@ import DA.Finance.Asset
 import DA.Finance.Asset.Settlement
 import DA.Finance.Types
 
-import DA.List
+import DA.List hiding (delete)
```

Comment:
woah, what does this do? haven't seen it before!

Reply:
`hiding` lets you import everything in a module minus the functions/templates/types you have listed there. The reason i did it here is because `DA.Next.Set` also has a `delete` function (which is the one I wanted to use) so there would be an ambiguous occurrence  error. The other way to go about this is to use `import qualified` for one of them.